lib/maths/CBoostedTree.cc

```diff
@@ -1154,14 +1177,23 @@ class CBoostedTree::CImpl final {
             result.emplace_back(std::log(m_Gamma / 10.0), std::log(10.0 * m_Gamma));
         }
         if (m_EtaOverride == boost::none) {
-            result.emplace_back(0.8, 1.0);
+            double rate{m_EtaGrowthRatePerTree - 1.0};
```

Comment:
Could you add a comment to explain what happens here (and all the magic numbers)?